resources/definitions/fdmprinter.def.json

```diff
@@ -3070,7 +3235,32 @@
                     "default_value": false,
                     "limit_to_extruder": "support_interface_extruder_nr",
                     "enabled": "support_enable",
-                    "settable_per_mesh": true
+                    "settable_per_mesh": true,
+                    "children":
+                    {
+                        "support_bottom_enable":
+                        {
+                            "label": "Enable Support Bottom",
+                            "description": "Generate a dense flooring below the support where it rests on the model. This will create a skin between the model and support.",
```

Comment:
The word 'flooring' feels really awkward to me. Please rephrase.

ALso I'd say the bottom is generated in the support, not under it, but I guess that's a matter of how you look at it.